packages/playwright/src/util.ts

```diff
@@ -304,7 +304,7 @@ const kExtLookups = new Map([
   ['.mjs', ['.mts']],
   ['', ['.js', '.ts', '.jsx', '.tsx', '.cjs', '.mjs', '.cts', '.mts']],
 ]);
-export function resolveImportSpecifierExtension(resolved: string): string | undefined {
+export function resolveImportSpecifierExtension(resolved: string, dontResolveDirectories: boolean): string | undefined {
```

Comment:
I'd reverse the option for easier understanding: `resolveDirectories: boolean`.

Reply:
done in https://github.com/microsoft/playwright/commit/8fe370acf8e4e802f00e640c7ba0eb62069bf5e4